styles/elements.less

```diff
@@ -14,3 +14,7 @@ crucible-currency {
     text-align: center;
   }
 }
+
+[data-action] {
```

Comment:
I worry about potential side effects of this rule, although it seems sensible. Out of some caution, let's scope this a bit more tightly to some selector on actor sheets specifically like `span.hex[data-action]` or similar.